projects/igniteui-angular/src/lib/input-group/input-group.component.ts

```diff
@@ -212,6 +222,21 @@ export class IgxInputGroupComponent extends DisplayDensityBase
         return this._type;
     }
 
+    /**
+     * Returns the theme of the `IgxInputGroupComponent`.
+     * The default is `material`.
+     * ```typescript
+     * @ViewChild("MyInputGroup")
+     * public inputGroup: IgxInputGroupComponent;
+     * ngAfterViewInit(){
+     *    let inputType = this.inputGroup.theme;
+     * }
+     * ```
+     */
+    public get theme(): IgxInputGroupTheme {
```

Comment:
Is this useful information that needs to be exposed to the user that should already know what theme they set?